app/src/androidTest/java/com/github/houseorganizer/houseorganizer/FirebaseTestsHelper.java

```diff
@@ -281,6 +283,48 @@ protected static void createTestShopList() throws ExecutionException, Interrupte
         shopList.setOnlineReference(t.getResult());
     }
 
+    /**
+     * This method will create a billsharer on Firestore
+     */
+    protected static void createTestBillsharer() throws ExecutionException, InterruptedException {
+        FirebaseFirestore db = FirebaseFirestore.getInstance();
+
+        // Store new shop list with one item for TEST_HOUSEHOLD_NAMES[0] on Firebase
+        DocumentReference household = db.collection("households")
+                .document(FirebaseTestsHelper.TEST_HOUSEHOLD_NAMES[0]);
```

Comment:
This is nitpicking but you can use directly TEST_HOUSEHOLD_NAMES[0] without FirebaseTestHelper since it is the same class

Reply:
done